src/main/resources/wayf_widget_template.js

```diff
@@ -17,38 +17,26 @@
 const WAYF_BASE_URL = "https://wayf-cloud-sandbox.literatumonline.com";
 
 const AUTHORIZATION_HEADER_KEY = "Authorization";
-const AUTHORIZATION_HEADER_VALUE = "Bearer @PUBLISHER_JWT@";
+const AUTHORIZATION_HEADER_VALUE = "@PUBLISHER_JWT@";
 
-const GLOBAL_ID_HEADER_NAME = "X-Device-Id";
```

Comment:
@balaf do we not need this because it's now in the cookie and the client does not need to understand the value or its location?